documentjs.json

```diff
@@ -1,8 +1,12 @@
 {
+  "siteDefaults": {
+    "debug": true,
+    "forceBuild": false
```

Comment:
@eduardojmatos if `forceBuild` was present even with `false` value documentjs will be run `Transpiling` and `Compiling` Template and Static files?